mailbox/api/src/main/java/org/apache/james/mailbox/model/MimePathImpl.java

```diff
@@ -52,17 +51,9 @@ public final int hashCode() {
         return Arrays.hashCode(positions);
     }
 
-    public String toString() {
-        final StringBuilder builder = new StringBuilder("MIMEPath:");
-        boolean isFirst = false;
-        for (int position : positions) {
-            if (isFirst) {
-                isFirst = false;
-            } else {
-                builder.append('.');
-            }
-            builder.append(position);
-        }
-        return builder.toString();
+    @Override
+    public final String toString() {
+        return "MIMEPath:"
+            + Arrays.toString(positions);
```

Comment:
the previous toString was much better as it's displaying a path dot-separated `a.b.c`